app/admin/admin_users.rb

```diff
@@ -0,0 +1,28 @@
+ActiveAdmin.register AdminUser do
```

Comment:
please change the file name to users.rb

Reply:
It is an auto-generated file used for different purpose